src/test/java/uk/co/ramp/covid/simulation/output/DailyStatsTest.java

```diff
@@ -205,6 +214,16 @@ public void testConsistentDeaths() {
                 totalAdditionalDeaths < totalHomeDeaths && totalAdditionalDeaths < totalHospitalDeaths);
         
         assertEquals("Inconsistent number of deaths", totalDeaths, actualDeaths);
+
+        // Test consistency in deathsByAge
+        Map<Integer, Integer> combinedDeathsByAge = new HashMap<>();
+        for (DailyStats s : stats.get(0)) {
+           s.deathsByAge.forEach((k, v) -> combinedDeathsByAge.merge(k, v, Integer::sum));
+        }
+
+        final int[] deathsByAge = {0};
```

Comment:
Why are you using an array here?

Reply:
Lambdas can't capture primitive types from outer scopes (since they aren't pass by reference) - I did try (for other tests) with an Integer also (but I think because they are treated like special primative variants that didn't work either). This was the refactoring intellij automatically suggested/applied